src/controller/data_model/controller-clusters.zap

```diff
@@ -20910,7 +20910,7 @@
               "side": "server",
               "type": "single",
               "included": 0,
-              "storageOption": "RAM",
+              "storageOption": "External",
```

Comment:
Are all these just coming from zap? I don't think they're correct. If these changes are just zap side effects, would you mind poking the zap channel to see what's up with that? As far as i can see, the AQ sensor only handles the AirQuality attribute as external, and the rest aren't handled.

Reply:
Yes, this is some zap side effect. I didn't touch AQ

Reply:
This attribute is forced to external in zcl.json, but when people did that they did not update this .zap file (e.g. by running zap-convert).  Note that the .matter file did not change here, because codegen was _already_ treating this as external.